examples/sdk-usage/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java'
```

Comment:
we should probably include the google java format plugin so we can get the examples to automatically match the rest of the codebase.